tests/tests.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+
+from unittest import TestCase
+import json
+
+from openfisca_web_api.app import create_app
+
+tester = create_app('openfisca_dummy_country').test_client()
+
+
+class ParametersRoute(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.response = tester.get('/parameters')
+
+    def test_return_code(self):
+        self.assertEqual(self.response.status_code, 200)
+
+    def test_headers(self):
+        headers = self.response.headers
+        self.assertEqual(headers.get('Country-Package'), 'openfisca-dummy-country')
+        self.assertEqual(headers.get('Country-Package-Version'), '0.1.1')
+
+    def test_item_content(self):
+        parameters = json.loads(self.response.data)
+        self.assertEqual(
+            parameters[u'impot.taux'],
+            { u'description': u'taux d\'impôt sur les salaires' }
+            )
+
+
+class ParameterRoute(TestCase):
+
+    def test_error_code_non_existing_parameter(self):
+        response = tester.get('/parameter/non.existing.parameter')
+        self.assertEqual(response.status_code, 404)
+
+    def test_return_code_existing_parameter(self):
+        response = tester.get('/parameter/impot.taux')
+        self.assertEqual(response.status_code, 200)
+
```

Comment:
You might consider adding also a test on:

- `/parameter`
- `/parameters/`
- `/parameter/`
- `/parameter/with-ÜNı©ød€`
- `/parameter/with%20url%20encoding`
- `/parameter/impot.taux/`
- `/parameter/impot.taux/too-much-nesting`
- `/parameter//impot.taux/`